src/guide/1/essentials/security/index.md

```diff
@@ -107,26 +107,32 @@ curl -X GET 'http://localhost:7512/?pretty'
 
 ## Defining Profiles
 
-A `profile` definition is a Javascript object that contains an array of policies, each composed of a roleId and an array of restrictions:
+A `profile` definition is a JSON object that contains an array of policies, each composed of a roleId and an array of restrictions:
 
 ```js
-const myProfileDefinition = {
-  policies: [
+{
+  "policies": [
     {
-      roleId: "< role Id >",
-      restrictedTo: [
+      "roleId": "<role identifier>",
+      "restrictedTo": [
         {
-          index: "< some index >",
-          collections: [
-            "< a collection >",
-            "< another collection >"
+          "index": "<an authorized index name>",
+          "collections": [
+            "<authorized collection 1>",
+            "<authorized collection 2>",
+            "<...>"
           ]
         },
-        ...
+        {
+          "index": "<another index name>"
+          // All collections are allowed
+        }
       ]  
     },
-    <another role>,
-    ...
+    {
+      "roleId": "<another role identifier>"
```

Comment:
nitpickint: I would put the example without restriction in the first place

Reply:
Agreed. I also applied this remark to the `restrictedTo` example.